src/components/pages/FeatureRequestPage.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import { Typography, Button } from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+import { pageContainer, pageBody } from './common/styles';
+import PageTitle from './common/PageTitle';
+
+const useStyles = makeStyles({
+  pageContainer,
+  pageBody,
+});
+
+export default function FeatureRequestPage() {
+  const [userHasAcknowledgedMessage, setUserHasAcknowledgeMessage] = useState(false);
```

Comment:
I don't think we will need these for the Feature Request page, as the acknowledgement message is only relevant for the Bug Report page.